File: app/src/main/java/com/moudle/app/InsetDrawableActivity.java
package com.moudle.app;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import android.util.Log;
import android.widget.ImageView;

/**
 * InsetDrawable        inset   将一张图插入到另外一张图
 */
public class InsetDrawableActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_inset);

        ImageView iv_inset = findViewById(R.id.iv_inset);

        Log.e("TAG", "InsetDrawableActivity.onCreate." + iv_inset.toString());

    }
}

File: app/src/main/java/com/moudle/app/activity/InsetDrawableActivity.kt
package com.moudle.app.activity

import android.os.Bundle
import android.util.Log
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.moudle.app.databinding.ActivityInsetBinding

/**
 * InsetDrawable        inset   将一张图插入到另外一张图
 */
class InsetDrawableActivity : AppCompatActivity() {
    private lateinit var insetBinding: ActivityInsetBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        insetBinding = ActivityInsetBinding.inflate(layoutInflater)
        setContentView(insetBinding.root)

        Log.e("TAG", "InsetDrawableActivity.onCreate. - ${insetBinding.ivInset}")
    }
}